corehq/apps/app_manager/views/forms.py

```diff
@@ -227,103 +227,111 @@ def _edit_form_attr(request, domain, app_id, unique_form_id, attr):
 
     """
 
-    app = get_app(domain, app_id)
-    form = app.get_form(unique_form_id)
```

Comment:
I'm guessing it's just this line that's failing - could we make the try/catch just around it?